deploy/configs/rules.yml

```diff
@@ -0,0 +1,28 @@
+groups:
+- name: oracle_keeper
+  rules:
+  # Alert for any instance that is unreachable for >2 minutes.
+  - alert: InstanceDown
```

Comment:
What's instance?

Reply:
Renamed to ExporterDown. We have two sets of rules, ExporterDown is triggered when the endpoint with the healthcheck page is not available, for example, when the service cannot start normally or is off, and KeeperDown OracleDown, which triggered when the healthcheck returns 0 within 2 minutes.